### PR TITLE
Added coveralls

### PR DESCRIPTION
**Pull request by nyashabryan**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
I think merge this branch before #14 so that the PRs are clearer

File: requirements.txt [hunk 1]
```diff
@@ -4,13 +4,18 @@ async-timeout==3.0.0
 attrs==18.1.0
 autobahn==18.6.1
 Automat==0.7.0
+certifi==2018.4.16
 channels==2.1.2
 channels-redis==2.2.1
+chardet==3.0.4
 constantly==15.1.0
 coverage==4.5.1
+coveralls==1.3.0
```
Comment (nathanbegbie) [inline requirements.txt]: It looks like in other projects, because coveralls is something we only use with Travis, we install it as part of the Travis set up - https://github.com/praekelt/molo/blob/develop/.travis.yml#L55

File: requirements.txt [hunk 2]
```diff
@@ -4,13 +4,18 @@ async-timeout==3.0.0
...
 daphne==2.2.0
 Django==2.0.6
+django-environ==0.4.5
```
Comment (nathanbegbie) [inline requirements.txt]: There seem to be some additions which are not part of this PR

**Review by nathanbegbie — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (nyashabryan) [reply requirements.txt]: This came with the `pip3 freeze` command which I had forgotten to do before the last pull request.

**Review by nathanbegbie — APPROVED** [review]
